specs/data_structures.md

```diff
@@ -29,45 +28,51 @@ Data Structures
-| `time`               | [Time](#time)             |             |
-| `lastBlockID`        | [BlockID](#blockid)       |             |
-| `lastCommitHash`     | [HashDigest](#hashdigest) |             |
-| `dataHash`           | [HashDigest](#hashdigest) |             |
-| `validatorsHash`     | [HashDigest](#hashdigest) |             |
-| `nextValidatorsHash` | [HashDigest](#hashdigest) |             |
-| `consensusHash`      | [HashDigest](#hashdigest) |             |
-| `appHash`            | [HashDigest](#hashdigest) |             |
-| `lastResultsHash`    | [HashDigest](#hashdigest) |             |
-| `evidenceHash`       | [HashDigest](#hashdigest) |             |
-| `proposerAddress`    | [Address](#address)       |             |
+| name                 | type                      | description                                       |
+| -------------------- | ------------------------- | ------------------------------------------------- |
+| `version`            | `uint64`                  | Version of the LazyLedger chain.                  |
+| `chainID`            | `uint64`                  | Chain ID. Each fork assigns itself a (unique) ID. |
+| `height`             | `uint64`                  | Block height. The genesis block is at height `1`. |
+| `time`               | [Time](#time)             |                                                   |
+| `lastBlockID`        | [BlockID](#blockid)       |                                                   |
+| `lastCommitHash`     | [HashDigest](#hashdigest) |                                                   |
+| `dataHash`           | [HashDigest](#hashdigest) |                                                   |
+| `validatorsHash`     | [HashDigest](#hashdigest) |                                                   |
+| `nextValidatorsHash` | [HashDigest](#hashdigest) |                                                   |
+| `consensusHash`      | [HashDigest](#hashdigest) |                                                   |
+| `appHash`            | [HashDigest](#hashdigest) |                                                   |
+| `lastResultsHash`    | [HashDigest](#hashdigest) |                                                   |
+| `evidenceHash`       | [HashDigest](#hashdigest) |                                                   |
+| `proposerAddress`    | [Address](#address)       |                                                   |
 
 ## Data
 
- | name  | type                            | description |
- | ----- | ------------------------------- | ----------- |
- | `txs` | [Transaction](#transaction)`[]` |             |
+Wrapper for transaction data, which is a simple list of [Transaction](#transaction)s.
+
+ | name  | type                            | description           |
+ | ----- | ------------------------------- | --------------------- |
+ | `txs` | [Transaction](#transaction)`[]` | List of transactions. |
```

Comment:
In the paper and in @musalbas' prototype the LL Transactions are called messages. Maybe we can stick to this in the spec here, too? I don't feel strongly about this but then we can have a clear distinction between Tx on a sidechain and Tx to LL layer1 Txs (aka messages).

Reply:
Agreed.

Reply:
To be clear, messages in my prototype are blobs of data. It seems here transactions are referring to transactions in the LazyLedger consensus-critical sidechain. In that case I think they should still be called transactions, and be clarified that these are LazyLedger "main sidechain" transactions (whatever we call that).

Reply:
Yes, here messages refers to things that change the consensus-critical state of LazyLedger, aka transactions for normal blockchains. In that context I guess "transactions" would be preferred.